### DISCUSSION TITLE
Skupper vs Istio vs others

### DISCUSSION THREAD

**Discussion by lucacalcaterra** [skupperproject/skupper#902] [Q&A] [answered]
I would like to know which alternatives are viable (others thans skupper) for my goal; i would split my services between home and frontend cloud (example nginx as frontend on cloud and local services on home cluster).

Think skupper is good enough but would to like others ways.

Thanks !

**Comment by grs** [comment, marked as answer]
A key difference is that istio would require that you open up your firewall to allow incoming tcp connections to your home cluster. Skupper does not require this.

**Reply by lucacalcaterra** [reply]
Thanks Very useful...  Liqo and others viable alternatives? SKupper is quite enough for me but for now ther isn't arm64 support on images

**Reply by grs** [reply]
Liqo also requires 'reciprocally accessible' endpoints, meaning you would again need to open up your firewall to external connections.

**Reply by lucacalcaterra** [reply]
> Liqo also requires 'reciprocally accessible' endpoints, meaning you would again need to open up your firewall to external connections.

so... long life to skupper :-) do you know when arm64 arm will build on quay ?
